lib/lotus/application.rb

```diff
@@ -37,6 +37,8 @@ def self.inherited(base)
       synchronize do
         applications.add(base)
       end
+
+      define_logger(base)
```

Comment:
@huydx We use to create those facilities in loader: https://github.com/lotus/lotus/blob/master/lib/lotus/loader.rb#L155

For instance, we define a `Bookshelf::Routes` constant which is an object that can be accessed everywhere from the application like we need to to with the logger.

If there isn't any specific need to keep this definition here, I would love to see it together with the other similar objects in `Loader`.

Technical note: Loader creates that classes when we instantiate the application (eg. `Bookshelf::Application.new`). So a logger is accessible when the application starts.